packages/design-system/src/components/Table/Table.jsx

```diff
@@ -144,7 +144,7 @@ export class Table extends React.PureComponent {
 Table.defaultProps = {
   scrollableNotice: (
     <Alert
-      className="ds-u-margin-y--1 ds-u-font-size--small ds-u-font-weight--normal"
+      className="ds-u-margin-y--1 ds-u-font-size--normal ds-u-font-weight--normal"
```

Comment:
the class `ds-u-font-size--normal` can be removed from the code as it doesn't exist

```suggestion
      className="ds-u-margin-y--1 ds-u-font-weight--normal"
```

Reply:
Thanks for picking that up.